src/integrationTest/java/uk/gov/hmcts/darts/common/controller/CourthouseApiTest.java

```diff
@@ -144,6 +153,39 @@ void courthousesWithRegionAndSecurityGroupsGet() throws Exception {
 
     }
 
+    @Test
+    @Transactional
+    void courthousesGetWithRegionId() throws Exception {
```

Comment:
Suggest renaming to `courthousesGet`, and renaming the existing `courthousesGet` on L85 to `adminCourthousesGet`.

Reply:
Updated

Reply:
updated